SemRush issues July 2022 - meta-descriptions v3 (#5453)
wazuh/wazuh-documentation: source/deployment-options/splunk/splunk-rbac.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::

>    :description: Create and map users in the Wazuh app for Splunk

Reviewer: ```suggestion
   :description: Create and map users in a Wazuh with Splunk installation.
```
